### PR TITLE
Mm modeling

### PR DESCRIPTION
**Pull request by mmmtnez**
Dear all,

As we said in our four previous pull requests, we divided the large model_building pull request in some smaller ones. This fifth pull request (mm_modeling), coming after the rigid fit of chimera (model_04_chimerafit), deals with the manual fitting/ refinement of an atomic structure regarding a density volume. mm_modeling contains the update of model_04_chimerafit, which introduced the rigid fitting of atomic structures to density volumes (with powerfit and chimera) and, in addition, includes the coot protocol to carry out the manual fitting/refinement.

All the protocols make intensive use of the "origin" added to volumes and consider the possibility of "origin" for atomic structures.

In detail:
[Points 1, 2, 3, 4 and 5 have been added according suggestions from Josue and Pablo]

1.- As powerfit protocol showed two viewers, we have deleted one of them (that was called "viewers.py") and we maintain the other one ("viewer_powerfit.py"). This last one allows the display of the fitting score ranking and the fitting itself with chimera.
2.- We have deleted the folder pyworkflow/em/utils that contained two sub-folders (CCP4 and chimera) containing respective ccp4-utilities.py and chimera-utilities.py.
3.- We have created the pyworkflow/em/convert_header that contains the sub-folder CCP4. The file convert.py, included in this sub-folder, contains methods to transform to and from  CCP4 headers. adaptBinFileToCCP4 method has been renamed as adaptFileToCCP4.
4.- We have created the folder pyworkflow/em/viewers that contains the chimera_utils.py file including several methods to visualize the results with chimera.
5.- We have removed REFMAC_BIN and -COOT from config.
6.- The atomic structures, as pdb and cif files, are  visualized (in chimera) with a coordinate system.
8.- A new dataset called  "model_building_tutorial" has been create.  This dataset contains files of atomic structures (pdb and cif files) and map volumes.
9.- Be aware that chimera is used in chimerafit protocol at two levels: 1) as tool for fitting, 2) to show (analyze) the result.
10.- An additional protocol, protocol_operate, has been included in the folder pyworkflow/em/packages/chimera. This new protocol, labeled in the SCIPION main menu in Tools/Claculators (chimera - chimera operate), allows to modify PDB files and save the corresponding results
11.- Tests that are not workflow tests have been moved to the new standard localization.
-----

Test related with the pull request

Test: scipion test tests.em.workflows.test_workflow_modeling.TestPowerFit
Test: scipion test tests.em.workflows.test_workflow_modeling.TestChimeraFit
Test: scipion test tests.em.workflows.test_workflow_modeling.TestCootRefinement
Test: scipion test em.packages.chimera.test_protocol_chimera_operate.TestChimeraOperate

Test: scipion test tests.em.protocols.test_protocols_import_volumes.TestImportVolumes
Test: scipion test em.packages.xmipp3.tests.test_protocol_extract_unit_cell.TestProtModelBuilding

### CODE REVIEW

**Comment by josuegbl** [comment]
There are two files ( data/tests/powerfit/3j79-pdb-bundle1.pdb and  data/tests/powerfit/ribosome_4.vol) that must be removed from PR.

File: config/templates/scipion.template [hunk 1]
```diff
@@ -22,6 +22,7 @@ EM_ROOT = software/em
 BSOFT_HOME = %(EM_ROOT)s/bsoft-1.9.0
 CTFFIND_HOME = %(EM_ROOT)s/ctffind-3.6
 CTFFIND4_HOME = %(EM_ROOT)s/ctffind4-4.1.8
+CCP4_HOME =  %(EM_ROOT)s/ccp4
```
Comment (josuegbl) [inline config/templates/scipion.template]: the standard in scipion is PACKAGE_HOME = %(EM_ROOT)/package-version.

File: pyworkflow/em/__init__.py [hunk 2]
```diff
@@ -26,23 +26,21 @@
 """
 This modules contains classes related with EM
 """
-import os
-import sys
+#import os
```
Comment (josuegbl) [inline pyworkflow/em/__init__.py]: remove commented code

File: pyworkflow/em/packages/ccp4/protocol_refmac.py [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+    REFMAC='refmac5'
+
+    def __init__(self, **kwargs):
+            EMProtocol.__init__(self, **kwargs)
+
+    #--------------------------- DEFINE param functions ----------------------
+    def _defineParams(self, form):
+        form.addSection(label='Input')
+
+        form.addParam('inputVolume', PointerParam, label="Input Volume", important=True,
```
Comment (josuegbl) [inline pyworkflow/em/packages/ccp4/protocol_refmac.py]: PEP8: lines too larges

File: pyworkflow/em/protocol/__init__.py [hunk 4]
```diff
@@ -50,5 +50,5 @@
 from protocol_picking_difference import ProtPickingDifference
 
 from monitors import *
-#from protocol_export import *
-from protocol_export.protocol_export_emdb import ProtExportEMDB
+from protocol_export import *
+#from protocol_export.protocol_export_emdb import ProtExportEMDB
```
Comment (josuegbl) [inline pyworkflow/em/protocol/__init__.py]: remove commented code

File: pyworkflow/em/protocol/protocol_import/base.py [hunk 5]
```diff
@@ -85,7 +85,8 @@ def _defineParams(self, form):
                            "'#' represents one digit that will be used as "
                            "micrograph ID\n\n"
                            "NOTE: wildcard characters ('*', '?', '#') "
-                           "cannot appear in the actual path.")
+                           "cannot appear in the actual path.\n\n"
+                           "File extensions accepted: .map, .mrc, .vol., .spi, .xmp")
```
Comment (josuegbl) [inline pyworkflow/em/protocol/protocol_import/base.py]: we should accept any volume regardless of its extension

File: config/templates/scipion.template (hunk 1 above)
Comment (mmmtnez) [reply config/templates/scipion.template]: done

File: pyworkflow/em/protocol/__init__.py (hunk 4 above)
Comment (mmmtnez) [reply pyworkflow/em/protocol/__init__.py]: done

File: pyworkflow/em/__init__.py (hunk 2 above)
Comment (mmmtnez) [reply pyworkflow/em/__init__.py]: done

Comment (mmmtnez) [reply pyworkflow/em/__init__.py]: done

**Pull request merged**